pkgs/applications/networking/feedreaders/feeds/default.nix

```diff
@@ -0,0 +1,108 @@
+{ lib
+, stdenv
+, fetchFromGitLab
+
+, appstream
+, gobject-introspection
+, meson
+, ninja
+, pkg-config
+, wrapGAppsHook
+
+, glib
+, gtk3
+, libhandy
+, webkitgtk
+, python3
+}:
+let
+  listparser = python3.pkgs.buildPythonPackage rec {
+    pname = "listparser";
+    version = "0.18";
+
+    src = python3.pkgs.fetchPypi {
+      inherit pname version;
+      sha256 = "0hdqs1mmayw1r8yla43hgb4d9y3zqs5483vgf8j9ygczkd2wrq2b";
+    };
+
+    propagatedBuildInputs = with python3.pkgs; [
+      requests
+      six
+    ];
+
+    doCheck = false;
+
+    meta = with lib; {
+      description = "A parser for subscription lists";
+      homepage = "https://github.com/kurtmckee/listparser";
+      license = licenses.lgpl3;
```

Comment:
The readme says `lgpl3Plus`

https://github.com/kurtmckee/listparser#what-is-listparser